[2024-12-04 05:55:21,058] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: KPIs_for_5_Minutes.fetch_master_Data 2024-12-04T05:55:02.914973+00:00 [queued]>
[2024-12-04 05:55:21,098] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: KPIs_for_5_Minutes.fetch_master_Data 2024-12-04T05:55:02.914973+00:00 [queued]>
[2024-12-04 05:55:21,100] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-12-04 05:55:21,100] {taskinstance.py:1088} INFO - Starting attempt 1 of 3
[2024-12-04 05:55:21,102] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-12-04 05:55:21,181] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): fetch_master_Data> on 2024-12-04T05:55:02.914973+00:00
[2024-12-04 05:55:21,187] {standard_task_runner.py:52} INFO - Started process 1268 to run task
[2024-12-04 05:55:21,192] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'KPIs_for_5_Minutes', 'fetch_master_Data', '2024-12-04T05:55:02.914973+00:00', '--job-id', '475', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/Kpi_dags/main.py', '--cfg-path', '/tmp/tmp3fdoo2br', '--error-file', '/tmp/tmph2ypb9w0']
[2024-12-04 05:55:21,194] {standard_task_runner.py:77} INFO - Job 475: Subtask fetch_master_Data
[2024-12-04 05:55:21,345] {logging_mixin.py:104} INFO - Running <TaskInstance: KPIs_for_5_Minutes.fetch_master_Data 2024-12-04T05:55:02.914973+00:00 [running]> on host ca28183b9a80
[2024-12-04 05:55:21,608] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Rule Manager
AIRFLOW_CTX_DAG_ID=KPIs_for_5_Minutes
AIRFLOW_CTX_TASK_ID=fetch_master_Data
AIRFLOW_CTX_EXECUTION_DATE=2024-12-04T05:55:02.914973+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-04T05:55:02.914973+00:00
[2024-12-04 05:55:21,610] {main.py:98} INFO - Fetching data from master db and storing excel between 2024-12-04 05:50:00 and 2024-12-04 05:55:00
[2024-12-04 05:55:21,611] {main.py:23} INFO - Connecting to database...
[2024-12-04 05:55:21,634] {base.py:69} INFO - Using connection to: id: master_table_conn. Host: 10.10.18.92, Port: 5432, Schema: landing_zone, Login: Chakradhar, Password: ***, extra: {}
[2024-12-04 05:55:21,909] {main.py:26} INFO - Connected successfully...
[2024-12-04 05:55:22,003] {main.py:106} INFO - data fetched successfully from master db and size is (0, 8)
[2024-12-04 05:55:22,004] {main.py:107} INFO - Empty DataFrame
Columns: [ts, tag_obj_id, asset_id, value, created_by, updated_by, create_time, updated_time]
Index: []
[2024-12-04 05:55:22,011] {main.py:109} INFO - data stored successfully
[2024-12-04 05:55:22,013] {python.py:151} INFO - Done. Returned value was: None
[2024-12-04 05:55:22,037] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=KPIs_for_5_Minutes, task_id=fetch_master_Data, execution_date=20241204T055502, start_date=20241204T055521, end_date=20241204T055522
[2024-12-04 05:55:22,168] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-04 05:55:22,190] {local_task_job.py:151} INFO - Task exited with return code 0
